index.html

```diff
@@ -10246,25 +10246,18 @@ <h3>Operating System Accessibility API mapping of ARIA enumerated attributes</h3
 			[[[#supportedState]]], and in [[[CORE-AAM]]].</p>
 		</section>
 		<section id="enumerated-attribute-values-html">
-			<h3>ARIA Enumerated Attributes in HTML</h3>
+			<h3>ARIA nullable DOMString Attributes in HTML</h3>
```

Comment:
Does this need to be limited to "in HTML"? HTML includes all the common use cases I can think of offhand (SVG, MathML, EPUB) but I don't know of a reason to limit this to just HTML and other technologies incorporated into HTML.